### PR TITLE
[PowerShellV2] Migrated to Node10

### PR DESCRIPTION
**Pull request by DaniilShmelev**
***Includes changes to auto-generated resources.resjson and task.loc.json that were missing in https://github.com/microsoft/azure-pipelines-tasks/pull/13122***

**Task name**: PowerShell

**Description**: changed execution handler to Node10; updated dependencies; removed typings; updated mocha; updated tasklib version

**Documentation changes required:** No

**Added unit tests:** No

**Attached related issue:** https://github.com/microsoft/build-task-team/issues/334

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

---

[Node6 backup branch](https://github.com/microsoft/azure-pipelines-tasks/tree/node6hotfixes/PowerShellV2)

### CODE REVIEW

File: Tasks/PowerShellV2/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -16,6 +16,8 @@
   "loc.input.help.errorActionPreference": "Prepends the line `$ErrorActionPreference = 'VALUE'` at the top of your script.",
   "loc.input.label.failOnStderr": "Fail on Standard Error",
   "loc.input.help.failOnStderr": "If this is true, this task will fail if any errors are written to the error pipeline, or if any data is written to the Standard Error stream. Otherwise the task will rely on the exit code to determine failure.",
+  "loc.input.label.showWarnings": "Show warnings as Azure DevOps warnings",
```
Comment (EzzhevNikita) [inline Tasks/PowerShellV2/Strings/resources.resjson/en-US/resources.resjson:19]: Is this changes related to migration?

File: Tasks/PowerShellV2/task.loc.json [hunk 2]
```diff
@@ -97,6 +97,15 @@
       "helpMarkDown": "ms-resource:loc.input.help.failOnStderr",
       "groupName": "advanced"
     },
+    {
+      "name": "showWarnings",
```
Comment (EzzhevNikita) [inline Tasks/PowerShellV2/task.loc.json:101]: The same question as above

File: Tasks/PowerShellV2/Strings/resources.resjson/en-US/resources.resjson (hunk 1 above)
Comment (DaniilShmelev) [reply Tasks/PowerShellV2/Strings/resources.resjson/en-US/resources.resjson:19]: no, it looks like in [this pr](https://github.com/microsoft/azure-pipelines-tasks/pull/13122) changes to task.loc weren't included, but they should have been

Comment (anatolybolshakov) [reply Tasks/PowerShellV2/Strings/resources.resjson/en-US/resources.resjson:19]: Let's update it also here since they haven't been included previously - @DaniilShmelev could you please mention it in PR description also?

**Review by anatolybolshakov — APPROVED** [review]

File: Tasks/PowerShellV2/Strings/resources.resjson/en-US/resources.resjson (hunk 1 above)
Comment (DaniilShmelev) [reply Tasks/PowerShellV2/Strings/resources.resjson/en-US/resources.resjson:19]: done!

**Review by EzzhevNikita — APPROVED** [review]

**Review by sdobrodeev — APPROVED** [review]

File: Tasks/PowerShellV2/task.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 2,
         "Minor": 179,
```
Comment (egor-bryzgalov) [inline Tasks/PowerShellV2/task.json:20]:
```suggestion
        "Minor": 180,
```

**Review by egor-bryzgalov — APPROVED** [review]

**Comment by maschmid-MSFT** [comment]
Started receiving a pipeline task timeout error with the Powershell task today and trying to figure out what's wrong with no luck so far. The task and script hasn't changed since I believe January and just now started failing. I know the script is running as I can see the output. At a certain point, there is a Wait-Event that is satisfied/triggered whenever a certain output string is written (we look for that string in output and then fire the event within the OutputDataReceived handler so our Wait-Event can continue. So it just sits there now on Wait-Event even though I can see the output that should fire the event. Anything I said ring a bell as far as this commit is concerned? Any ideas?

**Comment by anatolybolshakov** [comment]
Hi @maschmid-MSFT this seems to be related to issue https://github.com/microsoft/azure-pipelines-tasks/issues/14054 - could you please track progress about this one here?

**Comment by maschmid-MSFT** [comment]
Looks like a hotfix was released and verified that it does fix our issue. Quick turnaround. Thank you.
